webpack.config.js

```diff
@@ -0,0 +1,53 @@
+var path = require("path");
+var HtmlWebpackPlugin = require("html-webpack-plugin");
+var UglifyJsPlugin = require("uglifyjs-webpack-plugin");
+
+module.exports = {
+  mode: "development",
+  devtool: "eval",
+  cache: true,
+  entry: "./src/index.js",
+  output: {
+    filename: "[hash].bundle.js",
+    path: path.resolve(__dirname, "dist"),
+    publicPath: "/"
+  },
+  resolve: {
+    extensions: [".js", ".jsx"]
```

Comment:
should ".test.js" be add here as well

Reply:
No I don't think so. Any file ending in 'test.js' would be covered by the '.js' extension.

Reply:
ah yeah of course!